doc/classes/ArrayMesh.xml

```diff
@@ -6,21 +6,42 @@
 	<description>
 		The [ArrayMesh] is used to construct a [Mesh] by specifying the attributes as arrays.
 		The most basic example is the creation of a single triangle:
-		[codeblock]
+		[codeblocks]
+		[gdscript]
 		var vertices = PackedVector3Array()
 		vertices.push_back(Vector3(0, 1, 0))
 		vertices.push_back(Vector3(1, 0, 0))
 		vertices.push_back(Vector3(0, 0, 1))
-		# Initialize the ArrayMesh.
+
+		# Initialize the ArrayMesh
 		var arr_mesh = ArrayMesh.new()
 		var arrays = []
 		arrays.resize(ArrayMesh.ARRAY_MAX)
 		arrays[ArrayMesh.ARRAY_VERTEX] = vertices
-		# Create the Mesh.
+
+		# Create the Mesh
```

Comment:
```suggestion
		# Create the Mesh.
```
I'm not quite sure why you removed these periods.

Reply:
There were periods? Nonetheless, fixed.